src/events/request/RequestEventHandler.ts

```diff
@@ -78,6 +84,33 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 			}
 		}
 
+		const requestLimit = this.requestLimits.get( origin.channel.id );
+		const internalChannelId = this.internalChannels.get( origin.channel.id );
+		const internalChannel = await DiscordUtil.getChannel( internalChannelId );
+
+		if ( requestLimit && requestLimit >= 0 && internalChannel instanceof TextChannel ) {
+			const internalChannelUserMessages = internalChannel.messages.cache
+				.filter( message => message.embeds.length > 0 && message.embeds[0].author.name == origin.author.tag )
+				.filter( message => new Date().valueOf() - message.embeds[0].timestamp.valueOf() <= 86400000 );
+			if ( internalChannelUserMessages.size >= requestLimit ) {
+				try {
+					await origin.react( BotConfig.request.invalidTicketEmoji );
+				} catch ( error ) {
+					this.logger.error( error );
+				}
+
+				try {
+					const warning = await origin.channel.send( `${ origin.author }, you currently have created more requests than the daily limit for this channel, please wait until tomorrow before creating more requests.` );
```

Comment:
This message might be misleading. If we resolve the requests quickly, the user won't need to wait until tomorrow.

I'd prefer a less technical message, although I'm not quite sure how that would look.

Perhaps something like `<username>, you posted a lot of requests today that are still pending. Please wait for these requests to be resolved before posting more.`

Reply:
Ok, done. I think it looks fine.